pages/common/traefik.md

```diff
@@ -0,0 +1,19 @@
+# traefik
+
+> A HTTP reverse proxy and load balancer.
+
+- Start server with default config:
+
+`traefik`
+
+- Start server with custom config file:
```

Comment:
`with custom config file` -> `with a custom config file` might read better.

Reply:
As I stated in your other change, this was mirror to the Nginx page's wording. I'm open to changing it if you feel it's necessary.

Reply:
Here it reads as better English if it has an article before the noun "config file",  so I think this change is still needed. I'll update the nginx page too, if everyone else is ok with this?